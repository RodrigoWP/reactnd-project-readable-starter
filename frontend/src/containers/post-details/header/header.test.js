'use strict'

import React from 'react'
import { shallow } from 'enzyme'
import Header from '.'

it('Should render correctly', () => {
  const func = jest.fn()

  const component = (
    <Header
      postId='0'
      handleEditPost={func}
      handleRemovePost={func}
    />
  )

  expect(shallow(component)).toMatchSnapshot()
})
