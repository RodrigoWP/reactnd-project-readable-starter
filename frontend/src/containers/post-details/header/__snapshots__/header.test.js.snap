// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly 1`] = `
<div>
  <span />
  <span>
    Invalid date
  </span>
  <CrudMenu
    handleEdit={[Function]}
    handleRemove={[Function]}
    id="0"
  />
</div>
`;
